### DISCUSSION TITLE
Is it safe to drop prisma_migrate_shadow_db?

### DISCUSSION THREAD

**Discussion by mismaah** [prisma/prisma#17181] [Q&A] [answered]
There's about 50 of these prisma_migrate_shadow_db in my dev server. Is it ok to drop these?

**Comment by janpio** [comment, marked as answer]
Yes.

Usually those should be cleaned up by Prisma. What database and database provider are you using? Can you manually create and drop databases with the credentials you are using?

**Reply by mismaah** [reply]
Provider is postgresql. DB is PostgreSQL 13.1.

**Reply by janpio** [reply]
Interesting. How are you running or hosting this database?

**Reply by mismaah** [reply]
On-prem, on a VM.

**Reply by janpio** [reply]
Does the user you are using maybe have rights to create a new database, but not to drop it?

**Reply by mismaah** [reply]
There are many users and dbs plus many projects using prisma all connected to this Postgres instance so I'm finding it difficult to pinpoint which project/user is creating the shadow dbs.

**Reply by janpio** [reply]
I created a docs issue that we mention that it is safe to drop these when they are left in place for whatever reason: https://github.com/prisma/docs/issues/4390

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
